api/src/main/java/org/apache/iceberg/expressions/ExpressionVisitors.java

```diff
@@ -23,6 +23,9 @@
  * Utils for traversing {@link Expression expressions}.
  */
 public class ExpressionVisitors {
+
```

Comment:
Is this empty line required?

Reply:
I think so - the style guide expects a blank line between a class declaration and constructors + methods.